streams/src/main/java/org/apache/kafka/streams/processor/TaskId.java

```diff
@@ -76,13 +76,13 @@ public int partition() {
     /**
      * Experimental feature -- will return null
      */
-    public String namedTopology() {
-        return namedTopology;
+    public String topologyName() {
+        return topologyName;
```

Comment:
I am just curious why this was not a getter (ie getTopologyName()) instead?

Reply:
It's by convention I think. If you look at all of the other "getters" none of them are prefixed with get either. I.e. partition() or subtopology()